### ISSUE TITLE
Uncaught AttributeError: module 'os' has no attribute 'startfile'


### ISSUE THREAD

**Issue by baditaflorin** [MarcoDuiker/QGIS_QuickPrint#14]
Steps to reproduce
 1. First 
 2. Second 
 3. Third 

```
Traceback (most recent call last):
  File "/Users/florin/Library/Application Support/QGIS/QGIS3/profiles/default/python/plugins/QuickPrint/quick_print3.py", line 533, in run
    os.startfile(self.dlg.pdfFileNameBox.displayText())
AttributeError: module 'os' has no attribute 'startfile'

```
QuickPrint 3.6.0, QGIS 3.4.5-Madeira on Darwin 19.6.0

**Comment by MarcoDuiker** [comment]
If you are able to do so, please try Master of this repo. It should be fixed.

**Comment by MarcoDuiker** [comment]
Fixed in latest release 3.6.1.
